Sources/EpoxyLayoutGroups/Types/LayoutGroupUpdateAnimation.swift

```diff
@@ -0,0 +1,66 @@
+// Created by lindsay_pond on 5/13/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import UIKit
+
+public typealias GroupAnimationClosure = (_ animations: @escaping () -> Void, _ completion: @escaping (_ completed: Bool) -> Void) -> Void
```

Comment:
I think we can omit the "closure" suffix on this since it's duplicative with the type info and then add some documentation comments
```suggestion
/// A closure that's invoked to perform an animated update to the items in a layout group,
/// with an animations closure containing the updates to animate and a completion closure
/// to be invoked when the animations complete with a parameter of whether the animations
/// completed successfully.
public typealias GroupAnimation = (
  _ animations: @escaping () -> Void, 
  _ completion: @escaping (_ completed: Bool) -> Void) 
  -> Void
```